### DISCUSSION TITLE
Where does the $row variable comes on this of the docs?

### DISCUSSION THREAD

**Discussion by viniciusalvess** [Power-Components/livewire-powergrid#1157] [Q&A] [answered]
Hello,

I'm trying to add a dispatch to an button action I'm adding to my grid, but it is not clear where this $row variable comes from the [dipatch](https://v5.livewire-powergrid.com/table/row-actions-buttons.html#dispatch) section of the documentation.

when I try to use the example below, it gives me  the error:

###Error:
Undefined variable $row

###Code:
```php
Button::add('create-dish')  
    ->slot('Create a dish')
    ->class('bg-indigo-500 text-white')
    ->dispatch('postAdded', ['key' => $row->id']),
```

Thanks,
-Vin

**Comment by luanfreitasdev** [comment, marked as answer]
Hello!. You can add the $row parameter within actions and the current row item in the table will be loaded.

I fixed the documentation

![image](https://github.com/Power-Components/livewire-powergrid/assets/33601626/df12df8b-8ea6-4ea9-99e0-0338bfd562cf)

**Reply by viniciusalvess** [reply]
Thanks @luanfreitasdev I appreciate it!
